cob_hardware_emulation/scripts/emulation_follow_joint_trajectory.py

```diff
@@ -128,11 +133,11 @@ def fjta_cb(self, goal):
                 self.joint_states.position = point.positions
                 # set lower time bound for the next point
                 time_since_start_of_previous_point = latest_time_from_start
-            
+
             # set joint velocities to zero after the robot stopped moving (reaching final point of trajectory)
-            self.joint_states.velocity = [0.0] * len(self.joint_states.velocity)    
-            self.joint_states.effort   = [0.0] * len(self.joint_states.effort)
-            
+            self.joint_states.velocity = [0.0] * nr_points_dof
```

Comment:
This also ensures that the resulting count of velocities as well as efforts will always be equal to the number of dofs. Although, they are currently set to zero, doing that is more explicit than returning empty lists.
It could happen that
```python
len(self.joint_states.velocity) == 0 or len(self.joint_states.velocity) == 0
```
resulting in empty lists.